syft/pkg/cataloger/deb/cataloger.go

```diff
@@ -36,9 +36,9 @@ func (c *Cataloger) Name() string {
 }
 
 // Catalog is given an object to resolve file references and content, this function returns any discovered Packages after analyzing dpkg support files.
-// nolint:funlen
+
```

Comment:
This blank line could have very easily resulted from a [comment I made](https://github.com/anchore/grype/pull/442#discussion_r722612680) 😄  So I wanted to chime in — blank lines between a declaration and its doc comment is not something that's important to me. Also, I think it breaks the association of the documentation to the declared thing. I haven't tested this on pkg.go.dev, but I have tested it in GoLand, and I find that the blank line causes the "hover over" documentation to no longer include the doc comment text.